src/lando/jinja.py

```diff
@@ -72,15 +72,15 @@ def calculate_duration(start: str, end: Optional[str] = None) -> dict[str, int]:
     return {"minutes": int(result[0]), "seconds": int(result[1])}
 
 
-def tostatusbadgeclass(status: dict) -> str:
+def tostatusbadgeclass(landing_job: LandingJob) -> str:
     mapping = {
         "aborted": "Badge Badge--negative",
         "submitted": "Badge Badge--warning",
         "in_progress": "Badge Badge--warning",
         "landed": "Badge Badge--positive",
         "failed": "Badge Badge--negative",
     }
-    return mapping.get(status["status"].lower(), "Badge Badge--negative")
+    return mapping.get(landing_job.status.lower(), "Badge Badge--negative")
```

Comment:
Should we add some resiliency for https://bugzilla.mozilla.org/show_bug.cgi?id=1956970, e.g, if `landing_job.status == None`?